geoportailv3/static/js/draw/featurepopupdirective.js

```diff
@@ -111,8 +155,34 @@ app.FeaturePopupController.prototype.initForm_ = function() {
 app.FeaturePopupController.prototype.validateModifications = function() {
   this.feature.set('name', this.tempName);
   this.feature.set('description', this.tempDesc);
+  this.feature.dispatchEvent(app.DrawEventType.PROPERTYMODIFYEND);
```

Comment:
I personally don't like the idea of a directive relying on an `@enum` defined in an another directive. This is coupling this directive to the other.

As already said above I am questioning the need for a specific event type.